<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestDataProcessor</class>
 <widget class="QMainWindow" name="TestDataProcessor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Test Data Settings</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="subtaskbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Test Cases and Subtasks</string>
    </property>
    <widget class="QTreeWidget" name="SubtskTWID">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>411</width>
       <height>281</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <column>
      <property name="text">
       <string>Testdata</string>
      </property>
     </column>
    </widget>
    <widget class="QLineEdit" name="NCase">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Number of test cases:</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="NCaseSpBOX">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>41</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
    </widget>
    <widget class="QPushButton" name="CrSubtskBTN">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Create Subtask</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::ListAdd"/>
     </property>
    </widget>
    <widget class="QPushButton" name="RemSubtskBTN">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Subtask</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::ListRemove"/>
     </property>
    </widget>
    <widget class="QLineEdit" name="TotScore">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>251</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Total Score of Problem:</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="TotScoreTXT">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="AddCaseBTN">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Test Case</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::ListAdd"/>
     </property>
    </widget>
    <widget class="QPushButton" name="RemCaseBTN">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Test Case</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::ListRemove"/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="tdatainfobox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>581</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Test Data Information</string>
    </property>
    <widget class="QTabWidget" name="DataConfTWID">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>561</width>
       <height>311</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="validationbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>581</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Test Data Validation</string>
    </property>
    <widget class="QTextEdit" name="VerdictTXT">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="markdown">
      <string notr="true">N/A

</string>
     </property>
     <property name="html">
      <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#a9a9a9;&quot;&gt;N/A&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Verdict">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Verdict:</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="ValLog">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Log:</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="ValLogTXT">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>201</width>
       <height>131</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="RunValBTN">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="RefreshBTN">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::ViewRefresh"/>
     </property>
    </widget>
    <widget class="QLineEdit" name="FileContent">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>File Content:</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="FileContentTXT">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>201</width>
       <height>111</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="File">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>File:</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="FileTXT">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
